Robot.java

```diff
@@ -0,0 +1,109 @@
+ * project.
+ */
+public class Robot extends TimedRobot {
+  private VictorSPX mMotor;
+  private XboxController mController; 
+  private Compressor mCompressor;
+  private DoubleSolenoid mSolenoidOne;
+  private DoubleSolenoid mSolenoidTwo;
+  private int loopCounter;
+
+  /**
+   * This function is run when the robot is first started up and should be used for any
+   * initialization code.
+   */
+  @Override
+  public void robotInit() {
+    mMotor = new VictorSPX(2);
+    mController = new XboxController(0);
+    mCompressor = new Compressor(0);
+    mCompressor.setClosedLoopControl(true);
+  }
+
+  /**
+   * This function is called every robot packet, no matter the mode. Use this for items like
+   * diagnostics that you want ran during disabled, autonomous, teleoperated and test.
+   *
+   * <p>This runs after the mode specific periodic functions, but before LiveWindow and
+   * SmartDashboard integrated updating.
+   */
+  @Override
+  public void robotPeriodic() {}
+
+  /**
+   * This autonomous (along with the chooser code above) shows how to select between different
+   * autonomous modes using the dashboard. The sendable chooser code works with the Java
+   * SmartDashboard. If you prefer the LabVIEW Dashboard, remove all of the chooser code and
+   * uncomment the getString line to get the auto name from the text box below the Gyro
+   *
+   * <p>You can add additional auto modes by adding additional comparisons to the switch structure
+   * below with additional strings. If using the SendableChooser make sure to add them to the
+   * chooser code above as well.
+   */
+  @Override
+  public void autonomousInit() {
+    loopCounter = 0;
+  }
+
+  /** This function is called periodically during autonomous. */
+  @Override
+  public void autonomousPeriodic() {
+    if (loopCounter < 18) {
+      mSolenoidTwo.set(DoubleSolenoid.Value.kReverse);
+      mSolenoidTwo.set(DoubleSolenoid.Value.kForward);
+
+      loopCounter++;
+    }
+  }
+
+  /** This function is called once when teleop is enabled. */
+  @Override
+  public void teleopInit() {
+  }
+
+  /** This function is called periodically during operator control.
+   * If a certain key is pressed on the joystick, 
+   */
+  @Override
+  public void teleopPeriodic() {
+    mSolenoidOne = new DoubleSolenoid(1, 2);
+    mSolenoidOne = new DoubleSolenoid(1, 1, 2);
+
+    if (mController.getAButton()) {
+      mMotor.set(1);
+      mSolenoidOne.set(DoubleSolenoid.Value.kForward);
+    } else if (mController.getBButton()) {
+      mMotor.set(-1);
+      mSolenoidOne.set(DoubleSolenoid.Value.kReverse);
+    } else {
+      mMotor.set(0);
+      mSolenoidOne.set(DoubleSolenoid.Value.kOff);
+    }
+  }
+
+  /** This function is called once when the robot is disabled. */
+  @Override
+  public void disabledInit() {
+    mSolenoidOne.set(DoubleSolenoid.Value.kOff);
+  }
+
+  /** This function is called periodically when disabled. */
+  @Override
+  public void disabledPeriodic() {}
+}
```

Comment:
Could you please perform your changes on the `Robot.java` file? It is located within the project (e.g. [src/main/java/frc/robot/Robot.java](https://github.com/frc-6901/2021-Code/blob/master/src/main/java/frc/robot/Robot.java))